docs/src/constraints.md

```diff
@@ -608,17 +608,17 @@ The constraint `con` is now equivalent to `2x <= 2`.
 To modify the scalar coefficients of a cosntraint (but notably *not yet* the
```

Comment:
This wording about scalar coefficients of a constraint doesn't really make sense to me. I see why you were confused. What about:

To modify the linear coefficients of a variable in a constraint (but notably *not yet* the quadratic coefficients) ...

Reply:
That would make more sense to me.
I usually interpret scalar as in opposite to vector (or array).

Since it's the constraint not the coefficient that is linear, the following might be better still:

> To modify the coefficients on a variable in a linear constraint (but notably not yet the coefficients in a quadratic constraint) ...

Reply:
This method works for both linear and quadratic constraints. It modifies the coefficients on the variable in the linear part of the constraint:
```julia
model = Model()
@variable(model, x)
@constraint(model, con, 2x + x^2 <= 10)
set_standard_form_coefficient(con, x, 3.0)
julia> con
con : x² + 3 x ≤ 10.0
```

Open to suggestions if you have better wording.

Reply:
Good point. How about this:

> To modify the coefficients for a linear term in a constraint (but notably not yet the coefficients on a quadratic term) ...

Reply:
Sounds good

Reply:
@a-torgovitsky, do you mind modifying your pull request with the new wording? This way you'll receive the credit in the commit log for your suggestion.

Reply:
Ok, credit is good I suppose ;)